builtin/mainmenu/dlg_config_world.lua

```diff
@@ -109,12 +121,57 @@ local function get_formspec(data)
 	local with_error, enabled_mods_by_name = check_mod_configuration(data.worldspec.path, all_mods)
 
 	local mod = all_mods[data.selected_mod] or {name = ""}
+	local use_technical_names = core.settings:get_bool("show_technical_names")
+
+	-- Modern layout with consistent spacing and alignment
```

Comment:
I see no `real_coordinates[true]` or `formspec_version[n]` where `n` >= 2. I think this is probably to blame partly for the elements being all over the place?